src/github.ts

```diff
@@ -0,0 +1,78 @@
+import { summary } from "@actions/core";
+import { context, getOctokit } from "@actions/github";
+import { env } from "process";
+import { WranglerActionConfig } from "./wranglerAction";
+
+type Octokit = ReturnType<typeof getOctokit>;
+
+export async function createGitHubDeployment({
```

Comment:
Is there a test we could write to make sure we're able to successfully call these octokit methods when we should?